packages/react-native-win32/metro.config.js

```diff
@@ -4,13 +4,24 @@
 const fs = require('fs');
 const path = require('path');
 
+const rnwTesterPath = fs.realpathSync(
+  path.dirname(require.resolve('react-native-win32-tester/package.json')),
```

Comment:
No need to change, just food for thought: I've seen this pattern in a few places, would it make sense to have a shared metro config that takes care of the tester and rnw?

Reply:
There are some different enough semantics between configs that I think duplication is probably okay for now. We might want to start sharing more if we want to extract the resolver used by Playground though for use in other packages.